ninaddaudkhane-minneapolis.md

```diff
@@ -0,0 +1,7 @@
+First change in this file!!
+
+Places to eat:
+
+1) Spoon and Stable
+2) Houlihaans
+3) Natures Basket
```

Comment:
What is your favorite thing to eat at Natures Basket? I really like 🌮 s and 🌯 s, do they have good ones?